content/en/docs/Badges/MCH2022/software-development/_index.md

```diff
@@ -35,12 +35,15 @@ The other two plus (depending on how well you can count) :
   very unlikely we'll have the resource to help you fix the badge during the
   camp.
 - [RISC-V](risc-v) and
-  [Forth](https://github.com/badgeteam/mch2022-firmware-ice40/tree/master/projects/Forth)
-  because The Badge contains an FPGA, you can turn it into anything you want.
+  [Forth](https://github.com/badgeteam/mch2022-firmware-ice40/tree/master/projects/Forth):
+  Because the badge contains a FPGA, you can turn it into anything you want.
```

Comment:
it's "an" because it's eff pee gee ay  :)

sorry to be a smartass

Reply:
Pushed new commit with "an FPGA".